crates/cdk-mintd/Cargo.toml

```
@@ -37,6 +37,9 @@ url = "2.3"
 utoipa = { version = "4", optional = true }
 utoipa-swagger-ui = { version = "4", features = ["axum"], optional = true }
 rand = "0.8.5"
+chrono = "0.4.39"
+serde_json = "1.0.133"
+reqwest = "0.12.9"
```

```suggestion
chrono = "0.4.39"
serde_json = "1"
reqwest = "0.12"
```

Since for MSRV we need 24.3 of reqwest we should not set the patch version, cargo will use the lastest one we can. 

This change plus rebasing onto main should fix the MSRV CI errors